Build failure on i386 for v1.9

I'm trying to build v1.9 from source on i386, but `make` fails and I'm not sure why.

My golang version is 1.11.5. I'm building from source rather than using the standard prebuilt binary for i386 because my hardware is very old, and the prebuilt binary fails with an illegal instruction error most likely related to not having SSE2. I was able to build gitea up to v1.8 successfully using golang 1.10. The error is new as of v1.9.

I'm invoking Make like this:
`TAGS="bindata sqlite redis" make generate build`

It's failing like this:
```
github.com/chaseadamsio/goorgeous
Makefile:315: recipe for target 'gitea' failed
make: *** [gitea] Error 1
```
I can't tell whether `goorgeous` is where the failure happens, or if it's whatever comes after it.

I realize old hardware compatibility isn't exactly something to shed tears over, but I'd be interested in any ideas if this seems fixable.

--- a/Makefile
+++ b/Makefile
@@ -22,7 +22,7 @@ BINDATA := modules/{options,public,templates}/bindata.go
 GOFILES := $(shell find . -name "*.go" -type f ! -path "./vendor/*" ! -path "*/bindata.go")
 GOFMT ?= gofmt -s
 
-GOFLAGS := -i -v
+GOFLAGS := -v
 EXTRA_GOFLAGS ?=
 
 MAKE_VERSION := $(shell make -v | head -n 1)

